lib/apiservers/service/swagger.json

```diff
@@ -759,6 +759,9 @@
                 },
                 "user": {
                   "type": "string"
+                },
+                "grant_perms": {
```

Comment:
Let's just spell out "grant_permissions"; unlike the CLI, there's no worry about being too verbose, and we've generally managed to avoid abbreviating things in the API so far.